src/layer/ImageOverlay.js

```diff
@@ -193,8 +193,8 @@ export var ImageOverlay = Layer.extend({
 
 		// @event load: Event
 		// Fired when the ImageOverlay layer has loaded its image
-		img.onload = Util.bind(this.fire, this, 'load');
-		img.onerror = Util.bind(this._overlayOnError, this, 'error');

```

Comment:
Removed the `'error'` parameter here as the `_overlayOnError` method accepts no arguments.